### PR TITLE
[Enhancement] #108 Make job-opening#category/jobType an enumerable

### LINKED ISSUE #108: [Enhancement] Make job-opening#category/jobType an enumerable
I'd prefer to load this from a config file in igneous/config and mirror granite's enumerables.

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #108

### CODE REVIEW

**Comment by James1x0** [comment]
@JavaLavaMT Did you test this?

**Comment by James1x0** [comment]
@JavaLavaMT I've added a test for you. It's failing. Make it not do that.

**Comment by JavaLavaMT** [comment]
image

File: lib/models/job-opening/test.js [hunk 1]
```diff
@@ -181,6 +181,7 @@ describe('Unit :: Job Opening Model', () => {
       await jobOpeningData.save();
 
       let jobOpening = await JobOpening.findById(jobOpeningData._id);
+      console.log('job ', jobOpening);
```
Comment (James1x0) [inline lib/models/job-opening/test.js]: ✂️

File: lib/models/job-opening/index.js [hunk 2]
```diff
@@ -5,6 +5,7 @@
 const mongoose     = require('mongoose'),
       Schema       = mongoose.Schema,
       Relationship = Schema.ObjectId,
+      enumConfig  = require('../../../config/statics'),
```
Comment (James1x0) [inline lib/models/job-opening/index.js]: Can you realign these? And use destructuring for variables below...
`{ jobCategories, jobTypes } = require(...`

File: lib/models/job-opening/index.js [hunk 3]
```diff
@@ -25,7 +26,7 @@ var jobOpeningSchema = new Schema({
   name:        { type: String, required: true },
   title:       String,
   description: String,
-  category:    String,
+  category:    {type:String, enum: enumConfig.categories},
```
Comment (James1x0) [inline lib/models/job-opening/index.js]: Can you change categories to `jobCategories`? Add proper infix spacing as well:
`{ type: String, enum: ...` to L41 as well

**Pull request merged**